### ISSUE TITLE
Fake domain name registration emails.

### ISSUE THREAD

**Issue by xpufx** [wekan/wekan#5524]
I noticed you're listing a few emails you received as evidence that some company is trying to register your domain name. Those are scam emails from domain registering middle man. They are trying to scare you into protecting your domain name by registering it with them. They are the scammers, not the named entity in the email. There's no requirement for registrars to check the validity of the domain name claim. Anybody can register any domain, excepting local regulations where for example a .com.xx can only be registered by a company with the same name in the country of xx. As such, the domain registrar would not let you know that someone is trying to register your domain. They'll happily take their $20 and let you deal with the consequences. Just FYI.

**Comment by xet7** [comment]
@xpufx 

It was much more than $20, I can not afford to pay so much for domains that I don't use at all.

Those recent emails were from China registrar. To those I did reply, that those others, that would like to buy those wekan domains, can buy those domains.

Some years ago there was some original wekan domain, that expired, and some scammers bought it, made it a virus website. I tried to use my WeKan trandemark to get ownership of that domain to me, but it was not possible. That is why I have discontinued some social media accounts, because they had links to that old virus website. I also removed all social media account links from wekan github website, because I got some reports from security researchers that some of those social media sites had security problems and scams.
